xml/System.Activities.Presentation.PropertyEditing/EditModeSwitchButton.xml

```diff
@@ -100,7 +100,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a value that, when <see langword="true" />, means that the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> will be calculated automatically to match the <see cref="P:System.Activities.Presentation.PropertyEditing.PropertyContainer.ActiveEditMode" /> of the owning <see cref="T:System.Activities.Presentation.PropertyEditing.PropertyContainerEditMode" />. Otherwise, the mode to switch to will be based on the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> property.</summary>
+        <summary>Gets a value that, when <see langword="true" />, means that the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> will be calculated automatically to match the ActiveEditMode of the owning PropertyContainer. Otherwise, the mode to switch to will be based on the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> property.</summary>
```

Comment:
Unrelated to your PR: this summary should be succinct, and much of this description is more appropriate for the return value section. That's true of a number of other member descriptions for this type. This is a good issue to add to the community contributor's project.